### PR TITLE
Load/store-operations

### PR DESCRIPTION
**Pull request by cdaley95**
Implemented vary basic BasicML class with some inits, created load and store class methods.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -5,14 +5,36 @@
...
-  print("testing a feature specific 2")
-  return
+class BasicML:
+    '''Initialized with a 100 word memory and a single
+    register named accumulator, as well as a dictionary
+    for the opcodes'''
+    def __init__(self):
+        self.memory = ["+0000"]*100
+        self.accumulator = "+0000"
+        self.pointer = 0
```
Comment (zw49) [inline main.py:15]: What is this for?

**Review by zw49 — APPROVED** [review]
Looks good. What is that pointer variable for though?

File: main.py (hunk 1 above)
Comment (cdaley95) [reply main.py:15]: The pointer variable is for iterating the memory location. It starts at memory location 0. Currently, there isn't any function or method that uses it.

Comment (cdaley95) [reply main.py:15]: The branch methods will directly modify the pointer.

**Pull request merged**